.gitignore

```diff
@@ -12,6 +12,8 @@
 
 .env
 
+Pipfile
```

Comment:
This is unnecessary; we just shouldn't have this file in the top level directory at all.